pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -268,7 +269,9 @@ stdenv.mkDerivation rec {
     export TEST_TMPDIR=$(pwd)
 
     hello_test () {
-      $out/bin/bazel test --test_output=errors \
+      $out/bin/bazel test \
+        --test_output=errors \
+        --java_toolchain='${javaToolchain}' \
```

Comment:
Somewhat orthogonal question: should this nix derivation provide among its outputs a `.bazelrc` file ready to import into the user's own `.bazelrc`, setting up all these flags? They are needed anyway for compilations to succeed in a nix env.

Reply:
Hm, which flags would be interesting except for `--java_toolchain`?
The user codebases I know of are mostly rather large company repositories, with custom toolchains and everything.

Reply:
Sorry, I had in my mind all the other non-Java flags as well, like gcc and linker flags. But that's a separate issue.